src/AzureAuth.Test/AzureAuth.Test.csproj

```diff
@@ -23,8 +23,8 @@
 
   <ItemGroup>
     <!-- Stylecop required items -->
-    <AdditionalFiles Include="..\..\stylecop\stylecop.json" Link="stylecop.json" />
-    <Compile Include="..\..\stylecop\GlobalSupressions.cs" Link="GlobalSupressions.cs" />
+    <AdditionalFiles Include="..\stylecop\stylecop.json" Link="stylecop.json" />
+    <Compile Include="..\stylecop\GlobalSuppressions.cs" Link="GlobalSuppressions.cs" />
```

Comment:
```suggestion
    <Compile Include="..\stylecop\GlobalSuppressions.cs" Link="GlobalSuppressions.cs" />
    <Compile Include="..\stylecop\GlobalSuppressions.Test.cs" Link="GlobalSuppressions.Test.cs" />
```

Reply:
Should we change it in this PR or create a new one?

Reply:
This change was small enough Shalini and I thought it was okay :)